pip/operations/wheel.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import os
+
+from pip.exceptions import CommandError, PreviousBuildDirError
+from pip.index import PackageFinder
+from pip.req import InstallRequirement, RequirementSet, parse_requirements
+from pip.utils.build import BuildDirectory
+from pip.wheel import WheelBuilder
+
+
+logger = logging.getLogger(__name__)
+
+
+def build_wheel(index_urls, options, args, build_session_func):
```

Comment:
I don't like the idea of passing the `options` arg that is strongly linked to the command line...